transaction/core/src/blockchain/block_version.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2018-2021 The MobileCoin Foundation
+
+use core::{fmt, hash::Hash, ops::Deref};
+use mc_crypto_digestible::Digestible;
+use serde::{Deserialize, Serialize};
+
+/// A block version number
+#[derive(
+    Clone, Copy, Debug, Eq, PartialEq, PartialOrd, Ord, Serialize, Deserialize, Digestible, Hash,
```

Comment:
It's easier to check whether something is already implemented later when these are sorted...
```suggestion
    Clone, Copy, Debug, Deserialize, Digestible, Hash, Eq, Ord, PartialEq, PartialOrd, Serialize,
```

Reply:
I just copied the code from token id, if that's what you want let's fix botj